save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt

```diff
@@ -23,11 +28,21 @@ import kotlinx.coroutines.isActive
 
 val topBarUserField = topBarUserField()
 
+/**
+ * [Props] of the top bar user field component
+ */
+external interface TopBarUserFieldProps : PropsWithChildren {
```

Comment:
Any children used here? Take a look what are the differences between `Props` and `PropsWithChildren`

Reply:
renamed